docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3"
+services:
+  colouring-london:
+    container_name: colouring-london
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - postgres
+    environment:
+      DATABASE_URL: postgres://postgres:postgres@postgres:8080/colouring-london-db
+      NODE_ENV: development
+      PORT: 8080
+    ports:
+      - "8080:8080"
+  postgres:
+    container_name: postgres-colouring-london
+    image: postgres:14
+    ports:
+      - "5432:5432"
+    volumes: 
+      - ./postgresdata:/var/lib/postgresql/data
+      - ./colouring-london/init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
+    restart: always
+    environment:
+      POSTGRES_USER: dockeruser
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: colouring-london-db
+      APP_COOKIE_SECRET: 123456
```

Comment:
`APP_COOKIE_SECRET` should be an environment variable for the `colouring-london` node app container - this is probably causing the error @matkoniecz sees.

Reply:
fantastic, thanks Tom - one step closer 😄 :
![Screenshot 2022-02-09 at 09 50 39](https://user-images.githubusercontent.com/5486164/153171518-74e4f2c3-1687-45ff-b13f-625a7b9d7ead.png)